notebooks/heading_similarity.ipynb

```diff
@@ -0,0 +1,1668 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    plot += ylab('paper overlap fraction')
</pre><p>Should this be 'number of overlapping papers'?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Yep, forgot to update the label</p>